python/ray/_private/node.py

```diff
@@ -677,7 +676,9 @@ def dashboard_agent_listen_port(self):
     @property
     def dashboard_grpc_port(self):
         """Get the dashboard head grpc port"""
-        return self._dashboard_grpc_port
+        raise AttributeError(
+            "Dashboard grpc server is removed. There is no port to " "listen."
+        )
```

Comment:
Is this used? If not, we can completely remove it?

Reply:
Not used. I kept it for backward compatibility. I can remove it if you like.